lib/src/server/sse_handler.dart

```diff
@@ -97,7 +97,7 @@ class SseConnection extends StreamChannelMixin<String> {
         _sink.add('data: ${json.encode(data)}\n');
         _sink.add('\n');
         await _outgoingStreamQueue.next; // Consume from stream if no errors.
-      } catch (StateError) {
+      } catch (_) {
```

Comment:
I also cleaned this up but I don't know if the intention was `} on StateError catch (_) {`? Analyzer was flagging this.

I left the semantics identical with this change.

Reply:
The semantics should be on StateError, let's fix that now.

Reply:
done